src/modelfree/policies/loader.py

```diff
@@ -109,16 +109,23 @@ def load_random(path, env, env_name, index, transparent_params):
     return PolicyToModel(policy)
 
 
+# Lazy import for PPO1 and SAC, which have optional mpi dependency
 AGENT_LOADERS = {
     'zoo': load_zoo_agent,
-    'ppo1': load_stable_baselines(PPO1),
-    'ppo2': load_stable_baselines(PPO2),
-    'sac': load_stable_baselines(SAC),
+    'ppo2': load_stable_baselines(stable_baselines.PPO2),
     'old_ppo2': load_old_ppo2,
     'zero': load_zero,
     'random': load_random,
 }
 
+try:
+    # MPI algorithms -- only visible if mpi4py installed
+    from stable_baselines import PPO1, SAC
+    AGENT_LOADERS['ppo1'] = load_stable_baselines(PPO1)
+    AGENT_LOADERS['sac'] = load_stable_baselines(SAC)
+except ImportError:
+    pass
```

Comment:
It seems possibly good to log some kind of warning here to the effect of "MPI not available, so PPO1 and SAC won't be usable"; otherwise AFAICT it'll just hit a KeyError somewhere else down the line

Reply:
Fair point. I think warnings are easily missed so I've changed it to instead give an explicit error if PPO1 or SAC are used, explaining that MPI is the culprit.